server/mediaserver.go

```diff
@@ -848,11 +811,10 @@ func (s *LivepeerServer) HandlePush(w http.ResponseWriter, r *http.Request) {
 		ctx = clog.AddNonce(ctx, cxn.nonce)
 	}
 
```

Comment:
Maybe here we need redefine errorOut function.

Reply:
Sure, please give me some direction what could be improved?

Reply:
at line 829, used glog but now clog. so I think we should move function redefine here.
```suggestion
errorOut = func(status int, s string, params ...interface{}) {
		httpErr := fmt.Sprintf(s, params...)
		clog.Errorf(ctx, httpErr)
		http.Error(w, httpErr, status)
	}
```

Reply:
fixed